### DISCUSSION TITLE
Robots cannot share the damage

### DISCUSSION THREAD

**Discussion by damangguomoniqi** [trickerer/Trinity-Bots#365] [General]
Pet robots are immune to environmental damage such as lava，Temple of Ahn'Qiraj, Defender of Anubisath's skill Meteor The damage will not be shared by teammates

**Comment by trickerer** [comment]
Well, bots being immune to lava/drowning/falling/etc. seem more convenient than not. Maybe somebody else will have more to say on that.

If bot aren't affected by that skill AoE damage share mechanic, this discussion will be moved to issues.

**Reply by trickerer** [reply]
Fixed AoE in https://github.com/trickerer/Trinity-Bots/commit/9e2680bf00ce30bb8013690f00712b0a0492fd09
